File: src/main/java/loop/SumNumber.java
package loop;

import java.util.Scanner;

public class SumNumber {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = scanner.nextInt();
        scanner.close();

        int sum = 0;
        for (int i = 1; i <= number; i++) {
            sum += i;
        }

        System.out.println(sum);
    }
}

File: src/main/kotlin/loop/SumNumber.kt
package loop

import java.util.Scanner

fun main() {

    val scanner = Scanner(System.`in`)
    val number: Int = scanner.nextInt()
    scanner.close()

    var sum = 0
    for (i in 1..number) {
        sum += i
    }

    println(sum)

}